en/manage-ha-postgresql.md

```diff
@@ -0,0 +1,126 @@
+Title: PostgreSQL HA notes | MAAS
+
+
+# PostgreSQL HA: hot standby
+
+**Disclaimer: These are *notes* that the reader can peruse to get an idea of
+what may be involved at the command line level when setting up 'hot standby' HA
+mode in PostgreSQL. The below procedures are not regularly tested. Please use
+the
+[PostgreSQL documentation](https://www.postgresql.org/docs/9.5/static/high-availability.html)
+when setting up HA on PostgreSQL.**
+
+<!-- NOTES
```

Comment:
at the moment it will be served as part of the page - I have no idea if our search tools will distinguish it as such. Plus it leaves us open to accidentally including information we may not want to make public. I appreciate it is useful to have notes about things but maybe we should keep them elsewhere